test/box/net.box_incorrect_timeout_error_gh-6144.test.lua

```diff
@@ -0,0 +1,16 @@
+net = require('net.box')
+
+-- https://github.com/tarantool/tarantool/issues/6144
+
+_ = box.schema.space.create('test')
```

Comment:
You should drop this space at the end of test. Otherwise it will be visible for the rest tests launched after this one: if it's not specified explicitly, all tests are run on the same instance.

Reply:
The same concerns grants - you should revoke privileges.

Reply:
Fixed